reference/release-notes/release-notes.md

```diff
@@ -9,22 +9,23 @@ For instructions on how to update your Anbox Cloud deployment to later versions,
 
 | Release date   |  Release notes  |
 |----|----|
+| November 13 2024 | [Anbox Cloud 1.24.0](1.24.0.md) |
 | October 23 2024 | [Anbox Cloud 1.23.2 hotfix 1](1.23.2-hotfix1.md) |
 | October 16 2024 | [Anbox Cloud 1.23.2](1.23.2.md) |
 | September 18 2024 | [Anbox Cloud 1.23.1](1.23.1.md) |
 | August 14 2024 | [Anbox Cloud 1.23.0](1.23.0.md) |
 
 ## Upcoming release roadmap
 
-The current minor release is **1.23.0** and the next one will be **1.24.0**.
+The current minor release is **1.24.0** and the next one will be **1.25.0**.
 
 The following target dates for upcoming releases are not final and could vary depending on various factors such as availability of Android security patches. The release notes link will be updated on the day of the release.
 
 | Target date | Version | Planned updates |
 |----|----|----|
-| November 14 2024 | Anbox Cloud 1.24.0 | * VHAL improvements<br/> * Android 14 images<br/> * Support for additional upscaling algorithms in the streaming SDK<br/> * Android security updates for November 2024<br/> * Reworked charms<br/> * Bug fixes |
 | December 11 2024 | Anbox Cloud 1.24.1 | * Android security updates for December 2024<br/> * Bug fixes |
 | January 15 2025 | Anbox Cloud 1.24.2 | * Android security updates for January 2025<br/> * Bug fixes |
+| February 12 2025 | Anbox Cloud 1.25.0 | * Integration with the Canonical Observability Stack<br/> * Backup and restore support for the Anbox Cloud dashboard charm<br/> * Support for Ubuntu 24.04 (Noble Numbat) for all charms<br/> * Terraform plan replacing the Anbox Cloud bundles<br/> * Improvements to the Anbox Management Service (AMS)<br/> * Upgrade to LXD 5.21<br/> * Android security updates for February 2025<br/> * Bug fixes |
```

Comment:
@morphis This is from Jira, please confirm if we are okay with highlighting listed features for 1.25.

Reply:
@morphis @jat-canonical correct me if I'm mistaken, but as discussed in the sprint, the Terraform plan work has been moved from the February 2025 timeline to May 2025 (and we agreed that the TF work should be ready by early April).

Reply:
Yes @adglkh, Terraform shouldn't be in there. I am waiting for other feedback on this to push the changes together.

Reply:
We can keep it in here but label it as an early "alpha" release. The plan might be maintained independently of our regular releases.